src/android/Dgram.java

```diff
@@ -27,9 +27,17 @@
     private DatagramSocket datagramSocket;
     private DatagramSocketListener datagramSocketListener;
 
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
```

Comment:
It's probably okay, but sometimes people will call super after their own logic in onDestroy

Reply:
It looks like it won't impact us in this case:

https://stackoverflow.com/questions/4423671/why-does-super-ondestroy-in-java-android-goes-on-top-in-destructors